docs/basic_construction.md

```diff
@@ -0,0 +1,125 @@
+# Telescope - Basic Construction
+**ALBA** provides a method for a prover to convince a verifier that the prover knows at more than $n_f$ elements satisfying a predicate $R$, while only presenting a small subset of elements as proof. 
+This construction uses a recursive filtering process, guided by random oracles, to produce succinct and efficient proofs.
+
+## Overview
+- The prover builds a proof by creating tuples (sequences) of elements from $S_p$, the set of elements. 
+- The process begins with roots as starting points and expands them step-by-step. 
+- At each step, only tuples that satisfy specific conditions, as determined by a random oracle, are kept for further extension.
```

Comment:
We only check the tuple once it is updated. Perhaps you could write something like:
"At each step, the prover takes a tuple, extends it with an element of Sp, and perform an oracle check. If the check is successful, the tuple is kept, and may be extended further, otherwise it is discarded."